src/App.tsx

```diff
@@ -50,7 +55,11 @@ export const App: React.FC = () => {
           <div className="tile is-parent">
             <div className="tile is-child box is-success">
               <div className="block">
-                <UserSelector value={author} onChange={setAuthor} />
+                <UserSelector
```

Comment:
Verify that the `UserSelector` component is correctly implemented and receives the expected props.